diff --git a/node_modules/@ethersproject/networks/lib.esm/index.js b/node_modules/@ethersproject/networks/lib.esm/index.js
index 5c232b8..be0ee25 100644
--- a/node_modules/@ethersproject/networks/lib.esm/index.js
+++ b/node_modules/@ethersproject/networks/lib.esm/index.js
@@ -160,7 +160,11 @@ const networks = {
         name: "matic",
         _defaultProvider: ethDefaultProvider("matic")
     },
-    maticmum: { chainId: 80001, name: "maticmum" },
+    maticmum: { 
+        chainId: 80001, 
+        name: "maticmum", 
+        _defaultProvider: ethDefaultProvider("maticmum")
+    },
     optimism: {
         chainId: 10,
         name: "optimism",
diff --git a/node_modules/@ethersproject/networks/lib/index.js b/node_modules/@ethersproject/networks/lib/index.js
index 693a671..df38635 100644
--- a/node_modules/@ethersproject/networks/lib/index.js
+++ b/node_modules/@ethersproject/networks/lib/index.js
@@ -162,7 +162,11 @@ var networks = {
         name: "matic",
         _defaultProvider: ethDefaultProvider("matic")
     },
-    maticmum: { chainId: 80001, name: "maticmum" },
+    maticmum: { 
+        chainId: 80001, 
+        name: "maticmum", 
+        _defaultProvider: ethDefaultProvider("maticmum")
+    },
     optimism: {
         chainId: 10,
         name: "optimism",
